### PR TITLE
Mention mobile operating systems and add a requirement for User Agents to update muted when OS decides to mute/unmute

### PR DESCRIPTION
**Pull request by youennf**
***
<a href="https://pr-preview.s3.amazonaws.com/youennf/mediacapture-main/pull/838.html" title="Last updated on Dec 9, 2021, 2:34 PM UTC (0b63702)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-main/838/5af9608...youennf:0b63702.html" title="Last updated on Dec 9, 2021, 2:34 PM UTC (0b63702)">Diff</a>

### CODE REVIEW

**Comment by youennf** [comment]
Fixes https://github.com/w3c/mediacapture-main/issues/837

File: getusermedia.html [hunk 1]
```diff
@@ -887,14 +887,20 @@ <h4>Media Flow</h4>
...
-        <p>Muted is out of control for the application, but can be observed by
+        <p>Muted is out of control for the web application, but can be observed by
         the application by reading the {{MediaStreamTrack/muted}} attribute and listening
         to the associated events {{mute}} and {{unmute}}. There can be
         several reasons for a {{MediaStreamTrack}} to be muted:
         the user pushing a physical mute button on the microphone, the user
         closing a laptop lid with an embedded camera, the user toggling a
         control in the operating system, the user clicking a mute button in the
         User Agent chrome, the User Agent (on behalf of the user) mutes, etc.</p>
+        <p>On mobile operating systems, it is often the case that microphone access may
```
Comment (dontcallmedom) [inline getusermedia.html:898]:
```suggestion
        <p>On some operating systems, microphone access may
```

File: getusermedia.html [hunk 2]
```diff
@@ -887,14 +887,20 @@ <h4>Media Flow</h4>
...
+        get stolen from the User Agent when another native application gets access to it,
```
Comment (dontcallmedom) [inline getusermedia.html:899]:
```suggestion
        get stolen from the User Agent when another application with higher-audio priority gets access to it,
```

File: getusermedia.html [hunk 3]
```diff
@@ -887,14 +887,20 @@ <h4>Media Flow</h4>
...
+        for instance in case of an incoming phone call. The User Agent SHOULD provide
```
Comment (dontcallmedom) [inline getusermedia.html:900]:
```suggestion
        for instance in case of an incoming phone call on mobile OS. The User Agent SHOULD provide
```

**Review by dontcallmedom — COMMENTED** [review]
a few suggestions to make the wording less mobile-specific

**Review by jan-ivar — APPROVED** [review]

**Pull request merged**